Makefile

```diff
@@ -19,10 +19,10 @@ docker.start.bitcoin-core: ## Start bitcoin-core docker container
 		-rpcallowip=172.17.0.0/16 \
 		-rpcbind=0.0.0.0 \
 		-rpcuser=test \
-		-rpcpassword=test
+		-rpcpassword=test > /dev/null
 
 docker.stop: ## Stop docker containers
-	@docker stop $$(docker ps -q)
+	@docker stop $$(docker ps -q) > /dev/null
```

Comment:
This is kind of a hacky way to do this but it works exactly how we want. This is making me reconsider using something like [dockertest](https://github.com/ory/dockertest), but this is fine for now, no need to overcomplicate/overengineer this lol.

Reply:
I was imagining something like dockertest existed, that's cool to see. This is definitely pretty interesting/hacky lol but I agree it's works and it's pretty simple.